roles/patroni/final/tasks/init.yml

```diff
@@ -34,10 +30,14 @@
     and (patroni_cluster.members is not defined
       or patroni_cluster.members is empty)
```

Comment:
is there any chance `patroni_cluster.members` is defined and not empty in a first deploy?

Reply:
From what I can see, no.

* `patroni_cluster.members` is a list of members returned by `patronictl list` command
* `patroni_cluster.initialised` is only true if `patroni_cluster.members` is a list containing at least one item

In a first deploy there will be no members in the DCS, so `patroni_cluster.members` should be an empty list.